src/main/java/org/cactoos/text/EndsWith.java

```diff
@@ -62,6 +62,15 @@ public EndsWith(final Text origin, final Text suffix) {
         this.suffix = suffix;
     }
 
+    /**
+     * Ctor.
+     * @param origin The origin
+     * @param suffix The suffix
+     */
+    public EndsWith(final String origin, final String suffix) {
```

Comment:
@andreoss when you introduce a secondary constructor, please put it before the primary one, not after.